src/ReporterUtils.js

```diff
@@ -155,5 +165,6 @@ module.exports = {
     formatTest,
     formatTestTitle,
     formatSuiteTitle,
-    writeFile
+    writeFile,
+    hasSpecInTitle
```

Comment:
Provide a test to this new exported function (see test\specs\ReporterUtils.spec.js)